src/components/molecules/LoadingPage/LoadingPage.scss

```diff
@@ -0,0 +1,61 @@
+@import "scss/constants";
+
+.LoadingPage {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: $loading-page-background-color;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &__wrapper {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    gap: $loading-page-text-gap;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__logo-container {
+    position: relative;
+  }
+
+  &__playa {
+  }
```

Comment:
You can remove this and it's usages.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2005/commits/a172c44fbf91deb2b63b290bcdc476c820b80fc1 and https://github.com/sparkletown/sparkle/pull/2005/commits/813776a5c5144414214388b313bb19a368a87dfc